packages/api/src/graphql/query.private.ts

```diff
@@ -203,17 +203,25 @@ export const GraphQLQuery = new GraphQLObjectType<undefined, Context>({
       resolve(root, {id}, {authenticate, dbAdapter}) {
         const {roles} = authenticate()
         authorise(CanGetPeers, roles)
+
         return dbAdapter.peer.getPeers()
       }
     },
 
     peer: {
       type: GraphQLPeer,
       args: {id: {type: GraphQLNonNull(GraphQLID)}},
-      resolve(root, {id}, {authenticate, dbAdapter, loaders}) {
+      async resolve(root, {id}, {authenticate, dbAdapter, loaders}) {
         const {roles} = authenticate()
         authorise(CanGetPeer, roles)
-        return loaders.peer.load(id)
+
+        const peer = await loaders.peer.load(id)
+
+        if (peer?.isDisabled === true) {
```

Comment:
What's the reason we want `true` here and not just truthy?

```suggestion
        if (peer?.isDisabled) {
```